File: app/src/main/java/com/ryanrvldo/moviecatalogue/data/source/remote/response/ImageResponse.java
package com.ryanrvldo.moviecatalogue.data.source.remote.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.ryanrvldo.moviecatalogue.data.source.model.ImageItems;

import java.util.List;

import lombok.Data;

@Data
public class ImageResponse {

    @SerializedName("backdrops")
    @Expose
    private List<ImageItems> backdrops;
}

File: app/src/main/kotlin/com/ryanrvldo/moviecatalogue/data/remote/response/ImageResponse.kt
package com.ryanrvldo.moviecatalogue.data.remote.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.ryanrvldo.moviecatalogue.data.model.Image

data class ImageResponse(
    @Expose @SerializedName("backdrops") val backdrops: List<Image>
)